hack/update-estimator-protobuf.sh

```diff
@@ -36,10 +49,11 @@ ${GOPATH}/bin/go-to-protobuf \
 # The `go-to-protobuf` tool will modify all import proto files in vendor, so we should use go mod vendor to prevent.
 go mod vendor
 
-SERVICE_PROTO_FILES=$(find . -name "service.proto")
+SERVICE_PROTO_FILES=$(find . -not -path *./_tmp/* -name "service.proto")
 
 for file in ${SERVICE_PROTO_FILES[*]}; do
   protoc \
+    --proto_path=${KARMADA_ROOT} \
```

Comment:
just delete this proto_path, it works well. @Garrybest

Reply:
That's right!